WcaOnRails/app/views/competitions/_my_competitions_table.html.erb

```diff
@@ -16,26 +16,36 @@
     <table class="table competitions-table <%= past ? "table-striped" : "" %>">
       <thead>
         <tr>
+          <% if @show_registration_status && bookmarked %>
+            <th></th>
+          <% end %>
           <th><%= t 'competitions.adjacent_competitions.name' %></th>
           <th><%= t 'competitions.adjacent_competitions.location' %></th>
           <th><%= t 'competitions.adjacent_competitions.date' %></th>
           <th></th>
           <th></th>
           <th></th>
           <th><%= "Delegates" if show_delegates %></th>
+          <% if bookmarked %>
+            <th></th>
+          <% end %>
           <th class="big-column"></th>
         </tr>
       </thead>
       <tbody>
         <% competitions.each do |competition| %>
           <% registration = registrations_by_competition_id[competition.id] %>
+          <% bookmarked && opens_in_days = distance_of_time_in_words_to_now(competition.registration_open) %>
```

Comment:
this `some_boolean && assignment = var` is a very confusing idiom, please refrain from using it